Add Software carpentry link in Contributing.md (#21)
fatiando/contributing: CONTRIBUTING.md

* Aaron Meurer's [tutorial on the git workflow](http://www.asmeurer.com/git-workflow/)
* [How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)

If you're new to working with git, GitHub, and the Unix Shell, we recommend 

> starting with the [Software carpentry](https://software-carpentry.org/) lessons, 

Reviewer: ```suggestion
starting with the [Software Carpentry](https://software-carpentry.org/) lessons, 
```

* Aaron Meurer's [tutorial on the git workflow](http://www.asmeurer.com/git-workflow/)
* [How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)

> If you're new to git and Unix Shell, we recomen you the
> [Software carpentry](https://software-carpentry.org/) lessons, which are in
> English and Spanish:

Reviewer: ```suggestion
If you're new to working with git, GitHub, and the Unix Shell, we recommend 
starting with the [Software carpentry](https://software-carpentry.org/) lessons, 
which are available in English and Spanish:
```

> * [Version Control with Git](https://software-carpentry.org/lessons/) / :ceuta_melilla: [Control de
> versiones con Git](https://software-carpentry.org/lessons/)

Reviewer: ```suggestion
* :gb: [Version Control with Git](http://swcarpentry.github.io/git-novice/) / :es: [Control de
versiones con Git](https://swcarpentry.github.io/git-novice-es/)
```

> * [The Unix Shell](https://software-carpentry.org/lessons/) / :ceuta_melilla:
> [La Terminal de Unix](https://software-carpentry.org/lessons/)

Reviewer: ```suggestion
* :gb: [The Unix Shell](http://swcarpentry.github.io/shell-novice/) / :es:
[La Terminal de Unix](https://swcarpentry.github.io/shell-novice-es/)
```

> 
> 

Reviewer: ```suggestion
```
